### DISCUSSION TITLE
Baloon in a recyclerview item

### DISCUSSION THREAD

**Discussion by nfadhil13** [skydoves/Balloon#164] [Q&A]
![weird-behavior](https://user-images.githubusercontent.com/48423700/109662334-8700f180-7b9d-11eb-88c0-a2e8b596b61e.gif)

Hello, I really love your library, it helps me a lot. But I have a question for a specific implementation. So the case is : 

I want to trigger a balloon when I click a button in recyclerview items. So I'm using a listener to trigger the balloon. But somehow it's resulting weird behavior that you can see in the gif.

I hope you can help. Thanks for the response and for the lovely and easy library

**Comment by skydoves** [comment]
Hi, could you provide codes for building the Balloon?
And the Balloon is basically a new window over the old view,  so I recommend using the `focusable(true)` function. 
(Dismiss when users touch outside of the balloon)
